src/fileformats/file_format.h

```diff
@@ -243,7 +241,7 @@ class FileFormat
 	 * If the Importer can not be created, a FileFormatException shall be
 	 * thrown. The default implementation does just that.
 	 */
-	virtual std::unique_ptr<Importer> makeImporter(QIODevice* stream, Map *map, MapView *view) const;
+	virtual std::unique_ptr<Importer> makeImporter(const QString& path, Map *map, MapView *view) const;
```

Comment:
What is the semantics for both `map` and `view` being `nullptr`?

Reply:
None. I didn't invent it, I just started with the API of the Importer/Exporter classes. 
FileFormat and friends will need substantial review until they really can handle more then Map.